tests/e2e/hcp_cluster_test.go

```diff
@@ -0,0 +1,177 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/openshift/rosa/tests/ci/labels"
+	"github.com/openshift/rosa/tests/utils/config"
+	"github.com/openshift/rosa/tests/utils/exec/rosacli"
+)
+
+var _ = Describe("HCP cluster testing",
+	labels.Day2,
+	func() {
+		defer GinkgoRecover()
+
+		var (
+			clusterID      string
+			rosaClient     *rosacli.Client
+			clusterService rosacli.ClusterService
+			isHosted       bool
+		)
+
+		BeforeEach(func() {
+			By("Get the cluster")
+			clusterID = config.GetClusterID()
+			Expect(clusterID).ToNot(Equal(""), "ClusterID is required. Please export CLUSTER_ID")
+
+			By("Init the client")
+			rosaClient = rosacli.NewClient()
+			clusterService = rosaClient.Cluster
+			var err error
+			isHosted, err = clusterService.IsHostedCPCluster(clusterID)
+			Expect(err).ToNot(HaveOccurred())
+
+		})
+
+		AfterEach(func() {
+			By("Clean the cluster")
+			rosaClient.CleanResources(clusterID)
+		})
+
+		It("create and edit hosted-cp cluster with AuditLog Forwarding enabled/disabled via rosacli - [id:64491]",
+			labels.High, labels.NonClassicCluster,
+			func() {
+				By("Get cluster description")
+				output, err := clusterService.DescribeCluster(clusterID)
+				Expect(err).To(BeNil())
+				clusterDetail, err := clusterService.ReflectClusterDescription(output)
+				Expect(err).To(BeNil())
+
+				if !isHosted || clusterDetail.AuditLogForwarding != "Enabled" {
+					Skip("It does't non HCP cluster - cannot run this test")
```

Comment:
```suggestion
					Skip("It does't support on non HCP cluster - cannot run this test")
```

Reply:
I think `NonClassicCluster` has pointed you to use the HCP cluster. Besides, the message is incorrect, the case should only be run on the AuditLogArn HCP cluster.